assets/js/modules/forms.js

```diff
@@ -1,16 +1,17 @@
 import $ from 'jquery';
 import { trackEvent } from '../helpers/metrics';
 
+function handleBeforeUnload(e) {
+    // Message cannot be customised in Chrome 51+
+    // https://developers.google.com/web/updates/2016/04/chrome-51-deprecations?hl=en
+    trackEvent('Apply', 'Review step abandonment check', 'message shown');
```

Comment:
We might want to change the `Review step abandonment check` message to be something more generic now

Reply:
👍